programs/library-bridge/CatBoostLibraryHandlerFactory.cpp

```diff
@@ -0,0 +1,48 @@
+#include "CatBoostLibraryHandlerFactory.h"
```

Comment:
The code in this file is similar to https://clickhouse.com/codebrowser/ClickHouse/programs/library-bridge/SharedLibraryHandlerFactory.cpp.html